# HG changeset patch
# User Rik <rik@octave.org>
# Date 1514505369 28800
#      Thu Dec 28 15:56:09 2017 -0800
# Node ID e8d0573279b2b6943fb7d08d5072c4ff5d473ef0
# Parent  6ea279bbe94a57dcf7ed7fafc171c8cdcc8c4b02
maint: move xor.m from miscellaneous to general directory.

* scripts/general/xor.m: Moved from miscellaneous/ dir.

* scripts/general/module.mk, scripts/miscellaneous/module.mk:
Update build system.

diff --git a/scripts/general/module.mk b/scripts/general/module.mk
--- a/scripts/general/module.mk
+++ b/scripts/general/module.mk
@@ -80,17 +80,18 @@ FCN_FILE_DIRS += \
   %reldir%/shift.m \
   %reldir%/shiftdim.m \
   %reldir%/sortrows.m \
   %reldir%/sph2cart.m \
   %reldir%/structfun.m \
   %reldir%/subsindex.m \
   %reldir%/trapz.m \
   %reldir%/triplequad.m \
-  %reldir%/validateattributes.m
+  %reldir%/validateattributes.m \
+  %reldir%/xor.m
 
 %canon_reldir%dir = $(fcnfiledir)/general
 
 %canon_reldir%_DATA = $(%canon_reldir%_FCN_FILES)
 
 %canon_reldir%_privatedir = $(fcnfiledir)/general/private
 
 %canon_reldir%_private_DATA = $(%canon_reldir%_PRIVATE_FCN_FILES)
diff --git a/scripts/miscellaneous/xor.m b/scripts/general/xor.m
rename from scripts/miscellaneous/xor.m
rename to scripts/general/xor.m
diff --git a/scripts/miscellaneous/module.mk b/scripts/miscellaneous/module.mk
--- a/scripts/miscellaneous/module.mk
+++ b/scripts/miscellaneous/module.mk
@@ -61,17 +61,16 @@ FCN_FILE_DIRS += \
   %reldir%/tmpnam.m \
   %reldir%/unix.m \
   %reldir%/unpack.m \
   %reldir%/untar.m \
   %reldir%/unzip.m \
   %reldir%/ver.m \
   %reldir%/version.m \
   %reldir%/what.m \
-  %reldir%/xor.m \
   %reldir%/zip.m
 
 %canon_reldir%dir = $(fcnfiledir)/miscellaneous
 
 %canon_reldir%_DATA = $(%canon_reldir%_FCN_FILES)
 
 %canon_reldir%_privatedir = $(fcnfiledir)/miscellaneous/private
 
